virtool/validators.py

```diff
@@ -3,6 +3,7 @@
 import virtool.users.utils
```

Comment:
Can you please update this module with the following:
- tests for all validators
- docstrings and typing
- `from` imports